src/common/database.c

```diff
@@ -2728,17 +2745,25 @@ int _get_pragma_val(const struct dt_database_t *db, const char* pragma)
 
 void dt_database_maybe_maintenance(const struct dt_database_t *db, const gboolean has_gui, const gboolean closing_time)
 {
-  gboolean check_for_maintenance = FALSE;
-
   char *config = dt_conf_get_string("database/maintenance_check");
 
+  if(!g_strcmp0(config, "never")){
+    // early bail out on "never"
+    fprintf(stderr, "[db maintenance] please consider enabling database maintenance.\n");
```

Comment:
Can we consider this an error msg or debug msg? or general info msg like the one about creating default settings?